datatree/tests/test_datatree.py

```diff
@@ -693,3 +693,42 @@ def test_filter(self):
         )
         elders = simpsons.filter(lambda node: node["age"] > 18)
         dtt.assert_identical(elders, expected)
+
+
+class TestSubtreeMerge:
+    def test_merge_subtrees(self):
+        data_tree = xr.DataArray(
```

Comment:
The object your test creates here is not a `DataTree` object.

To create an actual `DataTree` object you should use either `DataTree.from_dict` or the `DataTree` constructor (i.e. the `__init__` method), as [described in the documentation here](https://xarray-datatree.readthedocs.io/en/latest/data-structures.html#creating-a-datatree).